src/MaaWpfGui/Res/Localizations/en-us.xaml

```diff
@@ -313,32 +313,32 @@ At the end of the level, enter "Normal/Hard" to switch between Normal and Tough
     <system:String x:Key="PR-D-1">Grd/Spc Ⅰ</system:String>
     <system:String x:Key="PR-D-2">Grd/Spc Ⅱ</system:String>
     <system:String x:Key="Annihilation">Annihilation</system:String>
-    <system:String x:Key="TodaysStageTip" xml:space="preserve">Today's Stage tips: </system:String>
+    <system:String x:Key="TodaysStageTip" xml:space="preserve">Today's open stages: </system:String>
     <system:String x:Key="CETip">CE (LMD)</system:String>
-    <system:String x:Key="APTip">AP (Shop Voucher)</system:String>
+    <system:String x:Key="APTip">AP (Purchase Certificate)</system:String>
     <system:String x:Key="CATip">CA (Skill Summary)</system:String>
     <system:String x:Key="LSTip">LS (Battle Record)</system:String>
     <system:String x:Key="SKTip">SK (Carbon)</system:String>
     <system:String x:Key="PR-ATip">PR-A (Med/Def Chip)</system:String>
     <system:String x:Key="PR-BTip">PR-B (Cst/Sni Chip)</system:String>
     <system:String x:Key="PR-CTip">PR-C (Pio/Sup Chip)</system:String>
     <system:String x:Key="PR-DTip">PR-D (Grd/Spc Chip)</system:String>
-    <system:String x:Key="Pormpt1">It's time to complete Annihilation</system:String>
-    <system:String x:Key="Pormpt2">Don't forget Annihilation</system:String>
+    <system:String x:Key="Pormpt1">It's Monday: time to clear the Annihilation</system:String>
+    <system:String x:Key="Pormpt2">It's Sunday: don't forget about Annihilation</system:String>
```

Comment:
This looks a bit long, have you tested if it displays properly? I'm not sure if the current control supports automatic line feeds

Reply:
I tested and found that had added auto line feeds before, so that's fine

Reply:
Perfect!